plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources.properties

```
@@ -224,6 +224,21 @@ dialog_scripts_import_wizard_name = Import script(s)
 dialog_scripts_import_wizard_title = Import script(s)
 dialog_scripts_import_wizard_window_title = Scripts Import Wizard
 
+dialog_workspace_import_wizard_window_title = Workspace Configuration Import Wizard
+dialog_workspace_import_wizard_window_restart_dialog_title = Restart {0}
+dialog_workspace_import_wizard_window_restart_dialog_message = You need to restart {0} to update the configuration.\nDo you want to restart?
```

I don't like repeating the word restart repetition here. Let's write the cause in the first sentence. For example, `The imported configuration was changed.\nDo you want to restart {0}?`
```suggestion
dialog_workspace_import_wizard_window_restart_dialog_message = You need to restart {0} to update the configuration.\nDo you want to restart?
```